EIPS/eip-2381.md

```diff
@@ -0,0 +1,146 @@
+---
+eip: 2381
+title: ENS support for ERC 721 TokenID
+author: Oisin Kyne (@OisinKyne) <oisin@kyne.eu>
+discussions-to: https://github.com/ethereum/EIPs/issues/2381
+status: Draft
+type: Meta
```

Comment:
```suggestion
type: Interface
```
Meta is for specifications about EIPs and the process itself.

Reply:
I think this is an ERC.

Reply:
I think @axic has it right. Changed to `ERC`.